### PR TITLE
Issue #2658 Making the default OpenShift version as  3.10.0

### PR DESCRIPTION
**Pull request by praveenkumar**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

Fixes <place holder for #issue number>

Steps to test the pull request

  1. 
  2. 
  3. 
  4.

### CODE REVIEW

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please.

File: test/integration/features/cmd-openshift.feature [hunk 1]
```diff
@@ -42,7 +42,7 @@ cluster in VM provided by Minishift.
        And stdout of command "minishift ssh -- "docker inspect --format={{.State.FinishedAt}} origin"" is not equal to "0001-01-01T00:00:00Z"
 
   Scenario: Pods docker-registry and router are ready after OpenShift restart
-     Given user waits "30" seconds
+     Given user waits "60" seconds
```
Comment (gbraad) [inline test/integration/features/cmd-openshift.feature]: :-/

File: test/integration/features/generic-driver.feature [hunk 2]
```diff
@@ -46,12 +46,11 @@ Feature: With generic driver Minishift can provision remote unprovisioned VM
      Then Minishift should have state "Running"
 
   Scenario: OpenShift configuration is not present on remote VM
-  NOTE: Directory openshift.local.volumes will be left at the place due to permision problems.
+  NOTE: Directory openshift.local.volumes will be left at the place due to permission problems.
```
Comment (gbraad) [inline test/integration/features/generic-driver.feature]: This change should be made nonetheless...

**Comment by praveenkumar** [comment]
retest this please

**Comment by gbraad** [comment]
why is the retest needed?

**Comment by praveenkumar** [comment]
> why is the retest needed?

@gbraad  So as per upstream issue, some of the failure bits should be not occurring every single time, I want to make sure what actually persistent.

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please

**Comment by praveenkumar** [comment]
Failure because of https://github.com/openshift/origin/issues/19300 but in our case only restart now failing :(

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please.

**Pull request merged**